README.md

```diff
@@ -42,29 +38,28 @@ you haven't.
 Each action's name when deployed on Adobe Runtime is `cla-` + the action name
 (subdirectory).
 
-Zip up the action
+#### Creating New Actions
+
+1. Zip up the action
 
 ```
 cd checker && zip -r checker.zip .
 ```
 
-If you haven't created a runtime action yet, run the following command:
+2. To create the action, run the following command:
```

Comment:
For lookup action we need an `npm install` command for installing the external csv-parse module.